### ISSUE TITLE
Large Training Loss

### ISSUE THREAD

**Issue by guoyd1999** [pengzhangzhi/Open-dLLM#8]
Hi,

We reproduce your pretraining code and find that the initial loss is about 29.7~28.0. Although it drops to 3.2 after 1.4k  training steps, the initial loss is still too high as the model is re-initialized. Could you explain this phenomenon? Thanks!

**Comment by pengzhangzhi** [comment]
ya this is normal. bc we add a new <mask> token to the vocab that the AR weight wasn't trained on.

**Comment by guoyd1999** [comment]
Could you let me know which file you edited to add a new token? We need a traditional AR-version experiment as the ablation study.
